docs/how-to/custom-env.txt

```diff
@@ -113,24 +112,24 @@ This can be configured in  your experiment configuration like below:
 Custom Docker Images in Determined
 ----------------------------------
 
-While our official images contain all the dependencies you need for basic
+While our official images contain all the dependencies needed for basic
 workloads, many workloads have extra dependencies. If those extra dependencies
-are quick to install, you may want to consider using the ``startup-hook.sh``.
-However, some dependencies take too long to install via the
+are quick to install, you may want to consider using ``startup-hook.sh``.
+However, some dependencies take too long to install via
```

Comment:
non-blocking: "For situations where installing dependencies via ``startup-hook.sh`` would take too long, we suggest building your own..."